src/v/cloud_roles/CMakeLists.txt

```diff
@@ -16,4 +16,7 @@ v_cc_library(
     v::http
     v::s3
     v::model
+    v::utils
```

Comment:
since nothing in the cloud roles library is changed in this commit it implies that this is either unnecessary, or required in a previous commit?

Reply:
I think this may be because I used `gate_guard` somewhere in the commit and removed it later. I will remove this dependency.

Reply:
this was due to `read_fully_to_string` which is in sts implementation. I have added the include there.